tests/unit/test_footprinter.cc

```diff
@@ -474,48 +515,15 @@ TEST_F(TestFootprinter, test_vector) {
       sizeof(v)
     );
   }
-}
-
-struct TestBase {
 
-  checkpoint_virtual_serialize_root()
-
-  virtual ~TestBase() = default;
-
-  template <typename SerializerT>
-  void serialize(SerializerT& s) {
-    s | a;
-  }
-
-private:
-  int a = 0;
-};
-
-struct TestDerived2 : TestBase {
-  explicit TestDerived2(int i) {}
-  explicit TestDerived2(SERIALIZE_CONSTRUCT_TAG) {}
-
-  checkpoint_virtual_serialize_derived_from(TestBase)
-
-  template <
-    typename SerializerT,
-    typename = std::enable_if_t<
-      std::is_same<
-        SerializerT,
-        checkpoint::Footprinter
-      >::value
-    >
-  >
-  void serialize(SerializerT& s) {
-    s | raw_pointer;
-    s | shared_pointer;
+  {
+    std::vector<TestDerived2> v;
+    EXPECT_EQ(
```

Comment:
Could this potentially return greater, if the default-constructed vector has a non-zero capacity?

Reply:
I was just testing that case (ad53ede) - it was too early to celebrate victory

Reply:
yes, this is implementation dependent, so I got lucky - changed to `GE`